### DISCUSSION TITLE
RF-source-6GHz (Discussions)

### DISCUSSION THREAD

**Discussion by aljazblatnik** [aljazblatnik/RF-source-6GHz#1] [Announcements]
## 👋 Hi!
  Please use Discussions as a place to connect with other members in hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Welcome others and are open-minded 💪.

  To get started, comment below!

**Comment by zxf123465** [comment]
I'm not sure exactly what design you are referring to. Usually, it's not recommended to simply go with the layout directly without thoroughly understanding the requirements and specifications. You might need to consider factors such as component availability, performance requirements, and potential compatibility issues. It's often a good idea to review and possibly make some adjustments based on your specific needs and the capabilities of your manufacturing process. So, it's better not to rely solely on the provided layout but to incorporate your own design and considerations for a more successful PCB.  Do you have any specific details or constraints about the signal source you want to build?

**Reply by aljazblatnik** [reply]
When designing the RF PCB, it's crucial to adhere to high-frequency rules. A microstrip circuit, specifically from the PLL output to the amplifier and attenuator, is the most straightforward approach in RF design. All connections should be 50 Ohm impedance. Other components should be placed as far from the lines as possible. Lines should be as straight as possible, with slight curves or 90-degree bends (RF tapered) as necessary. To accommodate the 50 Ohm line, a thinner laminate (less than 1.6 mm) is recommended to avoid excessively thick traces that could interfere with the 0.3 mm chip legs (impedance mismatch). Notably, the published design was successfully replicated using a homemade PCB circuit with toner transfer! But don't worry, minor deviations from the tolerances do not significantly affect the operation of the circuit.

**Comment by zxf123465** [comment]
Hello, I am a student and I would like to create a signal source based on your project. I was wondering if it is possible to directly follow your PCB layout for production?

**Reply by aljazblatnik** [reply]
I've added a new folder containing (beta) Gerber files for industrial manufacturing. Please note that these PCBs haven't been tested yet, but were produced at JLCPCB. The RF circuit is 0.6mm thick, double-sided FR4, while the remaining PCBs are 1.6mm thick, double-sided FR4. In my experience, FR4 circuits typically exhibit good repeatability up to around 8GHz, beyond which attenuation becomes significant.
